### DISCUSSION TITLE
Consider renaming the project

### DISCUSSION THREAD

**Discussion by m-sz** [mcmah309/rust#11] [General]
The name "rust" is very misleading, makes it hard to google for help regarding this particular library. Probably is in conflict with Rust foundation's trademark rules. [This reddit post](https://www.reddit.com/r/rust/comments/1h4wkyl/comment/m093uw5/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) by [u/zxyzyxz](https://www.reddit.com/user/zxyzyxz/) proposes a catchy name: `oxidart`.

**Comment by mcmah309** [comment]
It's not in conflict with the trademark. If you believe so, please cite the area you believe it may be so. `rust` is actually amply named since the api surface is almost exactly the same as the rust language. It's the third result for me when I google "dart rust package" and high up on pub.dev (for being relatively new, `rust_core` is still higher at the moment).

I would hate to rename again, unless it is needed.
